exept/exeptions.py

```diff
@@ -83,4 +83,19 @@ class UserAlreadyExistError(CustomQSportException):
     def __init__(self, date_value):
         self.date_value = date_value
         message = f"User with such data already exist'"
+        super().__init__(message)
+
+class IncorrectLogInStrategyMethod(CustomQSportException):
```

Comment:
Since it is an exception and not some usual class we have to specify that in name.

```suggestion
class IncorrectLogInStrategyMethodError(CustomQSportException):
```

Do tha for all exceptions you've made to keep it clean